src/encoding/src/base64.cairo

```diff
@@ -25,10 +26,126 @@ impl Base64UrlEncoder of Encoder<Array<u8>> {
         let mut char_set = get_base64_char_set();
         char_set.append('-');
         char_set.append('_');
-        inner_encode(data, char_set)
+        encode_u8_array(data, char_set)
     }
 }
 
+impl Base64FeltEncoder of Encoder<felt252> {
+    fn encode(data: felt252) -> Array<u8> {
+        let mut char_set = get_base64_char_set();
+        char_set.append('+');
+        char_set.append('/');
+        encode_felt(data, char_set)
+    }
+}
+
+impl Base64UrlFeltEncoder of Encoder<felt252> {
+    fn encode(data: felt252) -> Array<u8> {
+        let mut char_set = get_base64_char_set();
+        char_set.append('-');
+        char_set.append('_');
+        encode_felt(data, char_set)
+    }
+}
+
+fn encode_u8_array(mut bytes: Array<u8>, base64_chars: Array<u8>) -> Array<u8> {
```

Comment:
Suggest to move to a span as we're not modifying this ?
```suggestion
fn encode_u8_array(mut bytes: Array<u8>, base64_chars: Span<u8>) -> Array<u8> {
```

Reply:
fixed